index.ts

```diff
@@ -9,14 +9,32 @@ const bucketName: string = process.env.BUCKET_NAME as string;
 const getImage = async (event: EventDto): Promise<any> => {
 	try {
 		// parse params
+		const imageKey: string | undefined = event?.pathParameters?.imageName;
+		if (!imageKey) return {statusCode: 400, body: 'Image name is required'};
```

Comment:
consider using “!= null” for not null / not undefined check
it is clearer and safer than using “!" 
using “!” can cause issue if the imageKey are number